internalshared/configutil/listener.go

```diff
@@ -29,6 +29,7 @@ type ListenerTelemetry struct {
 	DisallowMetricsRaw              interface{}  `hcl:"disallow_metrics,alias:DisallowMetrics"`
 	MetricsOnly                     bool         `hcl:"-"`
 	MetricsOnlyRaw                  interface{}  `hcl:"metrics_only,alias:MetricsOnly"`
+	MetricsPath                     string       `hcl:"metrics_path,alias:MetricsPath"`
```

Comment:
@phil9909 In the context of OTEL paths, we'll probably add a `otel_metrics_path` and then later merge the two once otel takes over.